src/mlpack/methods/perceptron/perceptron_main.cpp

```diff
@@ -126,8 +133,13 @@ PARAM_MODEL_OUT(PerceptronModel, "output_model", "Output for trained perceptron"
 
 // Testing/classification parameters.
 PARAM_MATRIX_IN("test", "A matrix containing the test set.", "T");
+/* 
+* The PARAM_UROW_OUT("output") is deprecated and can be removed in mlpack4.0.0.
```

Comment:
Should we adjust the wording here, to 'will be removed in mlpack 4.0.0.'?

Reply:
Changes appropriately